README.md

```diff
@@ -1,9 +1,9 @@
 Picolarium
 ==========
 
-Picolarium is a [PICO-8](https://www.lexaloffle.com/pico-8.php) clone of the game _Polarium_ for DS.
+Picolarium is a [PICO-8](https://www.lexaloffle.com/pico-8.php) clone of the game _Polarium_ for DS and GBA.
```

Comment:
True enough! The reason I just wrote "DS" here is that the game doesn't contain any features from the GBA version (puzzles, extra tiles (#14), daily puzzle, etc).